app/validators/salary_validator.rb

```diff
@@ -7,15 +7,16 @@ class SalaryValidator < ActiveModel::EachValidator
 
   SALARY_FORMAT = /^(\d+|\d{1,3}(,\d{3})*)(\.\d{2})?$/.freeze
   MIN_SALARY_ALLOWED = 0
-  MAX_SALARY_ALLOWED = 200000
+  MAX_SALARY_LIMIT = 200000
 
   def validate_each(record, attribute, value)
```

Comment:
I wonder where did we call this method, I looked up but I couldn't find anything but the method. Is this overriding some rails validations? If so, is this used on any other fields than minimum salary fields?